packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -209,13 +230,24 @@ contract Attestations is
       );
     }
 
+    if(state.requests[msg.sender].blockNumber != 0) {
```

Comment:
That's an edge case I haven't thought of, great catch? I think effectively right now, this would mean that you can't request attestations for two different identifiers at any given time. I think that would be ok right now, but I assume you'll want a struct that would allows users to do so as it is the right thing to do?

Reply:
Nvm, requests is already indexed by identifier. I assume you are asking whether checking `mostRecentAttestationRequest` is the right one to use as the window, and it is not. Once https://github.com/celo-org/celo-monorepo/pull/1197/files gets merged, we would ensure that `block.number < state.requests[msg.sender].blockNumber + Random.randomnessBlockRetentionWindow`